self-hosted/Dockerfile

```diff
@@ -24,11 +23,11 @@ RUN set -x \
   && apt-get update && apt-get install -y --no-install-recommends $buildDeps \
   && rm -rf /var/lib/apt/lists/* \
   # grab gosu for easy step-down from root
-  && wget -O /usr/local/bin/gosu "https://github.com/tianon/gosu/releases/download/$GOSU_VERSION/gosu-amd64" \
+  && wget --quiet -O /usr/local/bin/gosu "https://github.com/tianon/gosu/releases/download/$GOSU_VERSION/gosu-amd64" \
```

Comment:
What's the reasoning behind adding `--quiet` here? Just minimize log spam?

Reply:
`wget` displays a progress bar which is meaningless in a docker build